integration-tests/src/test/groovy/com/okta/authn/sdk/its/AuthenticationClientIT.groovy

```diff
@@ -249,14 +249,14 @@ class AuthenticationClientIT extends AuthenticationTestSupport {
     void previousStateTest() {
         // first login to this group requires MFA enrollment
         def user = randomUser(null, TestConfiguration.CONFIG.mfaEnrollRequiredGroupId)
-        def response1 = authClient.authenticate(user.profile.getEmail(), USER_PASSWORD, null, ignoringStateHandler)
+        def response1 = authClient.authenticate(user.getProfile().getEmail(), USER_PASSWORD, null, ignoringStateHandler)
 
         assertThat response1.getStatus(), is(AuthenticationStatus.MFA_ENROLL)
         assertThat response1.getSessionToken(), nullValue()
 
         // enroll in TOTP factor
         def totpFactor = authClient.instantiate(TotpFactorProfile)
-                .setCredentialId(user.profile.getEmail())
+                .setCredentialId(user.getProfile().getEmail())
```

Comment:
Should this all be done in another PR?  I think you are doing 2 different things here?   1, changing the prop file, and then 2. this updating user.profile to be `getProfile`

Reply:
moved to #27, I'll update this PR after that one merges